201-blockchain-asaservice/azuredeploy.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "bcMemberName": {
+            "type": "String",
+             "metadata": {
+                "description": "Blockchain member name."
+            }
+        },
+         "memberPassword": {
+            "type": "secureString",
+            "metadata": {
+                "description": "Password for the BlockChain Administrator. The password must password must have 3 of the following: 1 lower case character, 1 upper case character, 1 number, and 1 special character that is not '#', '`', '*', '\"', ''', '-', '%',' ' or ';'."
+            }
+        },
+       "location": {  
+		"type": "string",  
+            "defaultValue": "[resourceGroup().location]",  
+            "metadata": {  
+                    "description": "Location for all resources."  
+            }  
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Blockchain/blockchainMembers",
+            "apiVersion": "2018-06-01-preview",
+            "name": "[parameters('bcMemberName')]",
+            "location": "[if(or(equals(parameters('location'),'southcentralus'),equals(parameters('location'),'southeastasia'),equals(parameters('location'),'westeurope'),equals(parameters('location'),'northeurope'),equals(parameters('location'),'westus2')),parameters('location'),'eastus')]",
```

Comment:
what problem are we trying to solve here?

Reply:
The idea was treat the available locations. But I will use the same aproach that we used with Machine Learning.